player.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
```

Comment:
This and the /scripts are what Alejandro put together for PIMS and what I'm familiar with. Open to modifying them or replacing them.